frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -54,15 +63,15 @@ export default {
   },
   computed: {
     ...mapGetters([
-      'machineTypesByCloudProfileNameAndZones',
-      'volumeTypesByCloudProfileNameAndZones',
+      'machineTypesByCloudProfileName',
+      'volumeTypesByCloudProfileName',
       'machineImagesByCloudProfileName'
     ]),
     machineTypes () {
-      return this.machineTypesByCloudProfileNameAndZones({ cloudProfileName: this.cloudProfileName })
+      return this.machineTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
     },
     volumeTypes () {
-      return this.volumeTypesByCloudProfileNameAndZones({ cloudProfileName: this.cloudProfileName })
+      return this.volumeTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
```

Comment:
volumeTypesByCloudProfileName does this function exist?

Reply:
done - merge error